README.md

```diff
@@ -1 +1,69 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] The Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly consider following this rule in future projects (You can skip it for this one to avoid the deletion of the branch)